src/main/resources/spark-swagger.conf

```diff
@@ -19,12 +19,14 @@ spark-swagger {
   # API related configs
 
   # Location where API docs will be published
-  host = "localhost:4567"
+  # (will be overwritten by PublicApiDocGenerator for the public-facing swagger document).
+  host = "AWS_API_SERVER"
 
   # basepath for calling the API, if applicable, otherwise leave as "/"
-  # as in https://example.api.com/<basepath>/user/all
-  basePath = "/"
-  docPath = "/doc"
+  # as in https://example.api.com<basepath>/user/all
+  # (will be overwritten by PublicApiDocGenerator for the public-facing swagger document).
+  basePath = "/AWS_API_STAGE"
```

Comment:
This change to a value other than `"/"` broke the endpoints. spark-swagger uses the initial value of this parameter to compose the final path of all endpoints.